### ISSUE TITLE
Scrolling without zooming, scrolling without PgDn

### ISSUE THREAD

**Issue by hackerb9** [ihabunek/toot#365]
I'm quite loving using `toot tui` with the awesome Sixel patch, but one problem I have is that images usually are too large and need to be scrolled. There are two problems I'm having with scrolling: First is that I can't seem to scroll a toot in the right side window unless I hit Z to zoom first. Am I missing something obvious? If not, may I suggest mapping <kbd>Spacebar</kbd> to scroll the toot contents even when a toot is _not_ zoomed? (This would be different from the PgDn key.)

Second, after zooming, I can scroll up and down using <kbd>j</kbd> and <kbd>k</kbd>, but it is painfully slow for large images. I'm using one of those little Apple Aluminum keyboards and it doesn't have convenient <kbd>PgUp</kbd> / <kbd>PgDn</kbd> keys. Again, I believe a good solution would be to allow the spacebar to scroll down and humbly suggest <kbd>Backspace</kbd> for scrolling back. If that is not possible, perhaps <kbd>shift</kbd>+<kbd>J</kbd> and <kbd>shift</kbd>+<kbd>K</kbd> would be logical choices.

Thanks!

**Comment by danschwarz** [comment]
right-arrow then scroll up/down with pgup/pgdown.  You don't get a focus indicator with that sixel patch as far as I remember.  My [super-deluxe experimental branch](https://github.com/danschwarz/toot/tree/everything3) has images, scrolling, and a focus indicator.  But it doesn't have sixel images, just kitty and iterm2 images 😢 .  That'll be added one day.
